src/main/resources/templates/db_scripts/migration_scripts/update_transformations_mapping_profile.sql

```diff
@@ -0,0 +1,35 @@
+--a Update fieldId fields in mapping profile transformations
+UPDATE ${myuniversity}_${mymodule}.mapping_profiles as profiles
+SET jsonb = jsonb_set(jsonb, '{transformations}', (
+  SELECT jsonb_agg(jsonb_set(transformations.value, '{fieldId}', to_jsonb(
+     CASE
+        WHEN transformations.value ->> 'recordType' = 'HOLDINGS' AND transformations.value ->> 'fieldId' NOT LIKE '%holdings.%'
```

Comment:
I think eletronic access link text will not be covered by this:
For EX: Previously it was "fieldId'": '"electronicAccess.linkText'
Now it is "fieldId" : "holdings.electronic.access.linktext.default",

Reply:
oh, yes, ok, will modify the script then